### PR TITLE
feat(sapphire): add spec

### PR DESCRIPTION
**Pull request by enxg**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Adds completion spec for the [Sapphire CLI](https://github.com/sapphiredev/cli)

### CODE REVIEW

File: src/sapphire.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    },
+    {
+      name: "help",
+      priority: 70,
+      description: "Display help for command",
+      displayName: "Display help for a command 📖",
+      args: {
+        name: "command",
+        description: "Command to display help for",
+        isOptional: true,
```
Comment (fedeci) [inline src/sapphire.ts:68]:
```suggestion
        isOptional: true,
        template: "help",
```

File: src/sapphire.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      },
+    },
+  ],
+  options: [
+    {
+      name: ["--help", "-h"],
+      priority: 10,
+      description: "Show help for sapphire",
```
Comment (fedeci) [inline src/sapphire.ts:76]: This allows this option to be inherited by all the subcommands. In this way we can remove the `--help` flags from there
```suggestion
      description: "Show help for sapphire",
      isPersistent: true,
```

File: src/sapphire.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      description: "Generates a component/piece",
+      displayName: "Generate a component/piece 📄",
+      args: [
+        {
+          name: "component",
+          description: "Component/piece name",
+        },
+        {
+          name: "name",
+          description: "File name",
```
Comment (fedeci) [inline src/sapphire.ts:38]:
```suggestion
          description: "File name",
          template: "filepaths",
          suggestCurrentToken: true,
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for this first PR @enxg! I left a couple comments to solve, let me know if you need any help!

**Comment by enxg** [comment]
> Thanks for this first PR @enxg! I left a couple comments to solve, let me know if you need any help!

All should be solved

**Comment by fedeci** [comment]
Great! Thanks again @enxg!

**Review by fedeci — APPROVED** [review]

**Pull request merged**